src/ChatHistoryItem.tsx

```diff
@@ -65,10 +78,8 @@ export const ChatHistoryItem = memo(
     const handleEdit = (e: React.MouseEvent) => {
       e.stopPropagation();
       if (editingTitle) {
-        setEditInputValue(title);
```

Comment:
These were unnecessary - the inputValue's initial value is already the title, and if it changes before the user closes, we have another handler to save that. This just toggles between editing and non editing state.

https://github.com/user-attachments/assets/7883ebb2-66bf-466e-9b12-10bec91e3060